src/main/scala/no/gdl/bookapi/integration/crowdin/CrowdinClientBuilder.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2018 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.integration.crowdin
+
+import com.typesafe.scalalogging.LazyLogging
+
+trait CrowdinClientBuilder {
+  val crowdinClientBuilder: CrowdinClientBuilder
+
+  class CrowdinClientBuilder extends LazyLogging {
```

Comment:
This class is not necessary for this pull-request, but will be needed in the next pull request where different crowdin-clients are used